libs/build_ecbuild.sh

```diff
@@ -35,11 +35,20 @@ URL="https://github.com/$repo/$name.git"
 [[ -d $software ]] && cd $software || ( echo "$software does not exist, ABORT!"; exit 1 )
 
 git checkout $version
+
+# Conform to NCO IT FISMA High Standards
+if [[ ${NCO_IT_CONFORMING:-"NO"} =~ [yYtT] ]]; then
+  sed -r -i 's|ssh://[a-zA-Z0-9.@]*||g' cmake/compat/ecmwf_git.cmake
+  sed -r -i 's|http[]://[a-zA-Z0-9@${}_./]*||g' cmake/compat/ecmwf_git.cmake
+  sed -r -i 's|http[]://[a-zA-Z0-9./-]*/test-data|DISABLED_BY_DEFAULT|g' cmake/ecbuild_check_urls.cmake
```

Comment:
I wonder if `ecbuild` has a macro (when building `ecbuild` itself) to do what lines 43 and 44 are doing.

Reply:
Lines 41 and 42 define `ECMWF_SSH` and `ECMWF_HTTPS` addresses to ECMWF internal git servers.  No one at NOAA (or external users) will ever have that.  Any attempt to access them will basically result in a timeout.  
I think these lines are for convenience in applications using `ecbuild` at ECMWF so users don't have to memorize them.  
Line 43 is benign, as one can provide the URL at the command-line to over-ride the `DISABLE_BY_DEFAULT`.
Removing that capability is far more intrusive.